.gitignore

```diff
@@ -3,3 +3,4 @@ coverage.html
 .idea
 .DS_Store
 docs
+package-lock.json
```

Comment:
package-lock.json should be kept in VCS as per https://docs.npmjs.com/configuring-npm/package-lock-json.html